OWML.ModHelper.Menus/ModInputCombinationPopup.cs

```diff
@@ -89,7 +89,7 @@ protected override void Update()
                 return;
             }
             if (currentlyPressedKeys.Count < 8 && (currentlyPressedKeys.Count > _combination.Count
-                || (currentlyPressedKeys.Count > 1 && _wasReleased)))
+                || currentlyPressedKeys.Count > 1 && _wasReleased))
```

Comment:
I wouldn't be so brave with bools in your place 🤔

Reply:
in this case it seems to be fine, but in some languages some boolean operators have same precedence tier